data/mods/Magiclysm/npc/trait_groups.json

```diff
@@ -135,5 +135,145 @@
       { "trait": "LARGE_OK_DRAGON", "prob": 50 },
       { "trait": "DRAGON_BREATH_BLACK", "prob": 50 }
     ]
+  },
+  {
+    "type": "trait_group",
+    "id": "NPC_starting_traits",
+    "//": "50% chance of an NPC to be an elf, dwarf, lizardfolk, etc.  It's a magical world.",
+    "subtype": "collection",
+    "traits": [ { "group": "trait_group_fantasy_species", "prob": 50 } ]
+  },
+  {
+    "//": "This group picks out one of the traits, leading to a 1% chance for an NPC to be a psion.",
```

Comment:
```suggestion
```
Pretty sure you want no comment here or at least a different one.

Reply:
Thank you for noticing that!